pkgs/development/libraries/pipewire/default.nix

```diff
@@ -6,7 +6,9 @@
 , python3
 , meson
 , ninja
+, eudev
 , systemd
+, enableSystemd ? stdenv.isLinux
```

Comment:
Does eudev work on something not linux?

Reply:
Not really. Technically, pipewire supports more platforms than both `systemd` and `eudev`, but that's out of scope for now I guess.
I changed the default to simply `true`, if someone has a need to be more restrictive (eg. `hostPlatform.isStatic` ) we can revisit this I'd say.